<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="OBSBasicPreview" name="preview" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnSourceImage">
           <property name="text">
            <string>添加图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSourceMonitor">
           <property name="text">
            <string>添加显示器</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSourceCamera">
           <property name="text">
            <string>添加摄像头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSourceVideo">
           <property name="text">
            <string>添加视频</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSourceText">
           <property name="text">
            <string>添加文字</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSourceAudioInput">
           <property name="text">
            <string>音频输入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSourceAudioOutput">
           <property name="text">
            <string>音频输出</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnStartPush">
           <property name="text">
            <string>开始推流</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStopPush">
           <property name="text">
            <string>停止推流</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelStreamInfo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OBSBasicPreview</class>
   <extends>QWidget</extends>
   <header>window-basic-preview.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
